### ISSUE TITLE
Frontend - add card functionality

### ISSUE THREAD

**Issue by wisskirchenj** [Hyperskill-Community/flashcards#82]
provide a component to allow the user to **add a new card** to a category.
The "add new card"-component should be accessible from the CardOverview component in the same way (+)-button as in the CategoryOverview page. But different from there, here it does not route away to another page, but replaces the component inside CardsPage to keep the state of the CardOverview.
- Add component should reuse the structure, logic and possibly subcomponents of `CardEdit`from #81.
- before displaying its CardType-dependent field layout, it should provide the user with a dropdown select component, to choose the intended CardType for the new card to add

This issue depends on #81 and should be implemented afterwards.

**Comment by wisskirchenj** [comment]
Completed with issues #94 #95 #96 
and #97
